examples/elasticsearch/README.md

```diff
@@ -0,0 +1,37 @@
+# Elasticsearch Example
+
+## Introduction
+[Elasticsearch](https://github.com/elastic/elasticsearch) is a widely used distributed database often used as a search engine or for logging. In this example, we demonstrate how to tune Elasticsearch using the benchmarking tool [Rally](https://esrally.readthedocs.io/en/stable/). Rally provides a variety of datasets, called tracks, that can be used to load test Elasticsearch. For each track there are several challenges, designed to test different workloads. This example shows how to i) use a setupTask to integrate with a Helm chart, and ii) set a maximum length a trial can run before being considered failed.
+
+## Prerequisites
+
+You must have a Kubernetes cluster. We recommend using a cluster with 3 nodes, 24vCPUs (8 on each node) and 48GB of memory (16 on each node). Additionally, you will need a local configured copy of `kubectl` and  to initialize Red Sky Ops in your cluster. You can download a binary for your platform from the [releases page](https://github.com/redskyops/k8s-experiment/releases) and run `redskyctl init` (while connected to your cluster). For more details, see [the installation guide](install.md).
+
+## Example Resources
+
+The resources for this tutorial can be found in the [`/examples/elasticsearch/`](https://github.com/redskyops/k8s-experiment/tree/master/examples/elasticseach) directory of the `k8s-experiment` source repository.
+
+* `experiment.yaml`
+: The actual experiment object manifest. In this experiment we vary the number of replicas for the Elasticsearch data nodes, and the CPU and memory assigned to each. The experiment aims to minimize both the time take to run the load tests and the cost of running the Elasticsearch cluster. The integration with Helm is in the setupTask in the trial template spec and in the patches section of the experiment spec. An empty patch tells the controller to wait until this object is ready before starting the trial. The maximum length a trial can run before being considered a failed trial is controlled by setting activeDeadlineSeconds in the pod spec in the trial template.
+
+* `service-account.yaml`
+: The manifest for the ServiceAccount used by the setupTask pod.
+
+* `rally-config.yaml`
+: The manifest for the ConfigMap used to configure Rally.
+
+To run the experiment use `kubectl` to first create the ConfigMap and ServiceAccount and then create the experiment.
```

Comment:
Does that mean we have 2 steps here, as opposed to the usual 1-step command to launch an experiment?
1. kubectl apply ...
2. kustomize build ...

Reply:
Yea. The kustomize workflow is a huge pain when you have a service account used by the experiment for setup tasks. It also helped me understand it better to manage the objects myself using kubectl rather than just hide everything behind kustomize

Reply:
I'm ok with that (@jgustie please weigh in on this as well). 

If we're keeping this I'd like to highlight the difference between this experiment and all the other ones - I can add that in the docs once we're done.

Reply:
I wouldn't bother highlighting it because it makes it look like kustomize is a requirement for the product, and in reality it's just a nice way to tie the simplest experiments into a single command. For more complicated experiments it has been getting in the way